templates/page.html

```diff
@@ -263,7 +264,7 @@ <h3>Developers</h3>
         <p>
           Understand REANA source code, adapt it to your needs, contribute changes back.
         </p>
-        <a href="https://github.com/reanahub/reana/wiki/">Developer Guide</a>
+        <a href="https://github.com/reanahub/reana/wiki/Development-installation">Developer Guide</a>
```

Comment:
It'll be better to link to the wiki home page as the entry point to the "Developer Guide", and rather amend the wiki so that the entry page would look like an index into the wealth of developer-related information.